model/Human-GEM.yml

```diff
@@ -172044,7 +172044,7 @@
           - MAM02751c: 1
       - lower_bound: 0
       - upper_bound: 1000
-      - gene_reaction_rule: "(ENSG00000101892 and ENSG00000137731 and ENSG00000163399) or (ENSG00000018625 and ENSG00000129244 and ENSG00000137731) or (ENSG00000018625 and ENSG00000069849 and ENSG00000137731) or (ENSG00000101892 and ENSG00000105409 and ENSG00000137731) or (ENSG00000105409 and ENSG00000137731 and ENSG00000143153) or (ENSG00000105409 and ENSG00000129244 and ENSG00000137731) or (ENSG00000069849 and ENSG00000105409 and ENSG00000137731) or (ENSG00000101892 and ENSG00000132681 and ENSG00000137731) or (ENSG00000132681 and ENSG00000137731 and ENSG00000143153) or (ENSG00000129244 and ENSG00000132681 and ENSG00000137731) or (ENSG00000069849 and ENSG00000132681 and ENSG00000137731) or (ENSG00000137731 and ENSG00000143153 and ENSG00000163399) or (ENSG00000129244 and ENSG00000137731 and ENSG00000163399) or (ENSG00000069849 and ENSG00000137731 and ENSG00000163399) or (ENSG00000018625 and ENSG00000101892 and ENSG00000137731) or (ENSG00000018625 and ENSG00000137731 and ENSG00000143153)"
```

Comment:
The first and you have removed here contains ENSG00000137731, which is not in the remove list in issue 444. I'm not sure if the other two are signaling enzymes that activates this enzyme, or how this works, but please check that this is correct.

Reply:
The same goes for ENSG00000163399. Looking this over is probably good.